docs/web-apps/automated-testing/playwright.md

```diff
@@ -20,7 +20,7 @@ Cucumber.js is not directly supported by Playwright. However, Playwright can be
 Supported OS:
 
 - Windows 10 / Windows 11
-- macOS 10.15+
+- macOS 11+
```

Comment:
This refers to the supported platform that saucectl runs on (client side!). _Not what's on Sauce Labs_.

1. Now, I personally don't care if you want to bump this too.
2. If this section is not clear enough with regards to what _System Requirements_ refers to, perhaps we should remove it altogether from this location 💭

Reply:
Ah, thanks for catching this. After your message yesterday I then realized I may have made the same mistake for the supported frameworks. I'll confirm on whether this section still makes sense for playwright and the other frameworks.

Reply:
Consensus was to stick to this change. thanks for highlighting!